PWGJE/Tasks/jetMatchedResponse.cxx

```diff
@@ -0,0 +1,392 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
```

Comment:
why is this in a separate task? You can combine it into the same task as the jetSpectraCharged. It is better not to separate the MC and data parts of tasks into multiple files

Reply:
Hi Nima, I just combine this matched task with the jetSpectraCharged. but I would like to keep several TH1 and TH2 for track and raw jet(before subtraction) to just confirm i was doing the right cut.